package.json

```
@@ -1,7 +1,11 @@
 {
   "name": "@saladtechnologies-oss/salad-cloud-transcription-sdk",
-  "version": "0.9.0-alpha.1",
+  "version": "0.9.0-alpha.2",
   "description": "The SaladCloud Transcription SDK. Please refer to the [SaladCloud API Documentation](https://docs.salad.com/api-reference) for more details.",
+  "main": "index.js",
```

Based on your build command, I think we should have all of these:

```suggestion
  "source": "./src/index.ts",
  "main": "./dist/index.js",
  "module": "./dist/index.mjs",
  "browser": "./dist/index.js",
  "unpkg": "./dist/index.js",
  "types": "./dist/index.d.ts",
```

---

Added